packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -299,6 +303,12 @@ export default class Calendar extends React.Component {
     }
   }
 
+  handleCloseButtonClick = (event) => {
+    if (this.props) {
```

Comment:
Did you mean to check for this.props.onRequestClose?

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/866/commits/14536d7f9271ab755e5e43fccd04bd10d492be09